apps/frontend/app/(client)/(main)/contest/[contestId]/@tabs/page.tsx

```diff
@@ -137,6 +196,26 @@ export default async function ContestTop({ params }: ContestTopProps) {
           )}
         </div>
       </div>
-    </>
+      <Accordion type="single" collapsible className="mt-16 w-[1208px]">
+        <AccordionItem value="item-1" className="border-b-0">
+          <AccordionTrigger className="w-[74px] border-t-[1.5px] border-[#a2a2a240] text-lg font-semibold">
+            More Description
+          </AccordionTrigger>
+          <AccordionContent className="text-base text-[#00000080]">
```

Comment:
피그마에 이부분 32px 정도의 간격이 있는데 pb-8 넣어주면 좋겠네요!
```suggestion
          <AccordionContent className="pb-8 text-base text-[#00000080]">
```

Reply:
수정했습니다~!